clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -736,7 +724,7 @@ protected void handleTagCommand() {
   }
 
   private String getOneTag(String[] tags) {
-    if (tags.length > 1) {
+    if (tags == null || tags.length > 1) {
```

Comment:
I don't think it is possible for tags to be null at this point, but I guess it doesn't hurt